package.json

```diff
@@ -123,6 +123,45 @@
         "command": "io.orta.jest.coverage.toggle",
         "title": "Jest: Toggle Coverage Overlay"
       }
+    ],
+    "debuggers": [
+      {
+        "type": "vscode-jest-tests",
+        "label": "Debug Jest tests using vscode-jest",
+        "languages": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
+        "configurationSnippets": [
+          {
+            "label": "Jest: Default jest configuration",
+            "description": "Running tests by directly running jest",
+            "body": {
+              "type": "node",
+              "name": "vscode-jest-tests",
+              "request": "launch",
+              "program": "^\"\\${workspaceFolder}/node_modules/jest/bin/jest\"",
+              "args": ["--runInBand"],
+              "cwd": "${workspaceFolder}",
```

Comment:
Oh, I overlooked that since it also works without a `cwd`. However including the `cwd` has the advantage of better directing the user if he has to change paths.
Debugging of `react-scripts-ts` seems to be not working at the moment with newer versions of jest ((wmonk/#276)[https://github.com/wmonk/create-react-app-typescript/issues/276]).